src/main/java/sirius/biz/storage/layer2/BlobDispatcher.java

```diff
@@ -157,21 +153,48 @@ public DispatchDecision dispatch(WebContext request) throws Exception {
         Values uriParts = Values.of(uri.split("/"));
         String type = uriParts.at(0).asString();
         if (Strings.areEqual(type, PHYSICAL_DELIVERY)) {
-            String filename = stripAdditionalText(uriParts.at(3).asString());
-            physicalDelivery(request,
-                             uriParts.at(1).asString(),
-                             uriParts.at(2).asString(),
-                             Files.getFilenameWithoutExtension(filename),
-                             filename);
+            if (uriParts.length() == 4) {
```

Comment:
would be nice of such changes would be an extra commit with an explaining commit message